helm/kubernetes-cluster-prep/bin/get-conjur-cert.sh

```diff
@@ -110,6 +111,14 @@ function main() {
   if [[ "$internal_addr" = false ]]; then
     echo "Assuming Conjur instance is outside of the Kubernetes cluster."
 
+    openssl_ver="$(openssl version)"
```

Comment:
If you combine this line with the next with a pipeline logical OR:
```
    status=0
    openssl_ver="$(openssl version) || status="$?"
    if [ "$status" != 0 ] ; then
      echo "OpenSSL is required to gather the SSL certificate"
      exit 1
    fi
    echo "Using " "$openssl_ver"
```
then we'd be able to use the `set -e` flag for the script.

Reply:
ok, that wrks, thanks!